build.xml

```diff
@@ -298,6 +299,16 @@
     <echo>Server binary ${server.bin}</echo>
     <echo>Server version ${server.version}</echo>
     <echo>Test Type(unit|ci) ${test.type}</echo>
+    <if>
+      <equals arg1="${server.cert}" arg2="not specified"/>
+      <then>
+        <echo>Servers start with TLS disabled</echo>
```

Comment:
Is this the first argument, or the second one?

Reply:
This is when the first argument equals to the second argument